VERSION

```diff
@@ -82,13 +82,13 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=12:6:1
+libmpi_so_version=12:7:0
 libmpi_cxx_so_version=2:3:1
-libmpi_mpifh_so_version=12:1:1
-libmpi_usempi_tkr_so_version=6:0:1
+libmpi_mpifh_so_version=12:1:0
+libmpi_usempi_tkr_so_version=7:0:2
 libmpi_usempi_ignore_tkr_so_version=8:0:2
 libmpi_usempif08_so_version=12:1:1
-libopen_rte_so_version=12:5:0
+libopen_rte_so_version=12:4:0
```

Comment:
@jsquyres There was a change made to the ORTE library - why did you revert the change to the revision number?
```
# - If the library source code has changed at all since the last
#   update, then increment revision (`c:r:a' becomes `c:r+1:a').
```